Scoop/Controller/ProfileViewController.swift

```diff
@@ -240,7 +245,7 @@ class ProfileViewController: UIViewController {
         }
         
         let musicLabel = UILabel()
```

Comment:
You're creating a lot of labels with essentially the same setup params...I know in SwiftUI I used to create a lot of custom classes so I didn't have to repeat all the alignment, font, etc, and this would be just `musicView.addSubview(CustomUILabel("No Music"))`.

I don't think you actually need to update this here or now, but in the future, if you ever find yourself creating a lot of elements with the same characteristics, I'd go this route. 

I'm not sure if this is bad advice, so @ReadeP, agree?

Reply:
Agreed. You could add these custom views to a "Supporting Views" folder.

Reply:
Ohhh good idea. I'll add a TODO for the future, but idk if it will be addressed in our first submission